client/src/urlHelpers.js

```diff
@@ -0,0 +1,46 @@
+
+
+var getCheckInOrOutDateFromUrl = (searchPath, lookFor) => {
+  //lookFor should either be 'checkIn' or 'checkOut' depending on which date you want
+  var searchParams = {};
+  var searches = searchPath.split(/[=?&]/);
+  for (var i = 1; i < searches.length; i = i + 2) {
+    searchParams[searches[i]] = searches[i + 1];
+  }
+  if (searchParams[lookFor === 'checkIn' ? 'check_in' : 'check_out'] === undefined) {
```

Comment:
It seems like this is here because you need to translate `checkIn` to `check_in`? Wondering if there's a way to avoid this by making the property name the same.

Reply:
Yeah... I did this because almost everywhere else in the code I use 'checkIn' style but airbnb puts check_in in the URL, so I thought I'd avoid mistakes in the rest of the code and isolate the airbnb style to here. It does look silly though I agree.